parent/pom.xml

```diff
@@ -102,9 +102,13 @@
     <plugin.version.dependency>3.1.1</plugin.version.dependency>
     <plugin.version.spotbugs>3.1.12.2</plugin.version.spotbugs>
     <plugin.version.sonar>3.7.0.1746</plugin.version.sonar>
+    <plugin.version.clirr>2.8</plugin.version.clirr>
 
     <!-- maven extensions -->
     <extension.version.os-maven-plugin>1.6.2</extension.version.os-maven-plugin>
+    
+    <!-- version against which backwards compatibility is checked -->
+    <backwards.compat.version>0.22.0</backwards.compat.version>
```

Comment:
Will this be updated when we release, or do we need to update this manually? I assume the latter?

Reply:
Thanks for pointing this out. Please check the latest commit, when releasing the version will be updated to the release version